data/core/step1_get_started.md

```diff
@@ -300,6 +319,35 @@
     - utter_core_tutorial
     - utter_anything_else
 
+## skip to info on rasa x
+* how_to_get_started{"product": "x"}
+    - utter_x_tutorial
+    - utter_anything_else
+
+## skip to info on rasa
+* how_to_get_started{"product": "rasa"}
+    - utter_explain_nlu
+    - utter_explain_core
+
+## skip to info on rasa nlu
+* how_to_get_started{"product": "nlu"}
+    - utter_ask_for_nlu_specifics
+* nlu_info{"nlu_part": "intent classification"}
+    - utter_nlu_intent_tutorial
+    - utter_offer_recommendation
+
+## not new to rasa + x
+* how_to_get_started
+    - utter_getstarted
+    - utter_first_bot_with_rasa
+* deny
```

Comment:
is the path for affirm here handled somewhere?

Reply:
Fixed - 04ab9c5